extkeys/hdkey_test.go

```diff
@@ -111,6 +111,21 @@ func TestBIP32Vectors(t *testing.T) {
 			"xpub6FnCn6nSzZAw5Tw7cgR9bi15UV96gLZhjDstkXXxvCLsUXBGXPdSnLFbdpq8p9HmGsApME5hQTZ3emM2rnY5agb9rXpVGyy3bdW6EEgAtqt",
 			"xprvA2nrNbFZABcdryreWet9Ea4LvTJcGsqrMzxHx98MMrotbir7yrKCEXw7nadnHM8Dq38EGfSh6dqA9QWTyefMLEcBYJUuekgW4BYPJcr9E7j",
 		},
+		// Test vector 3
```

Comment:
can you reference where this came from for future folks looking at this?

Reply:
good point @corpetty, I'm adding it to the PR description, it's from here:

https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#test-vector-3

Reply:
@mandrigin I added more comments in the code taken from the PR description

Reply:
hey @mandrigin @corpetty is there anything else you want to change in this PR?